metals/src/main/scala/scala/meta/internal/metals/ClassPath.scala

```diff
@@ -0,0 +1,35 @@
+package scala.meta.internal.metals
+import scala.meta.internal.semanticdb.SymbolOccurrence
+
+object ClassPath {
```

Comment:
Can we give this a more specific name? For example: `TypeSignature.scala` with `TypeSignature.fromSymbolOccurrence`. There is already a `scala.meta.io.Classpath`, which is a wrapper around a JVM classpath such as `a.jar:/user/b/classes`

Reply:
You are right, let's avoid the confusion. Also, Classpath doesn't reflect the intention behind this object very well. In the end, I went with `JvmSignatures` to distinguish it form other kinds (for example semanticdb signatures)